docs/components/connectors/out-of-the-box-connectors/github-webhook.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: github-webhook
-title: GitHub Webhook
-sidebar_label: GitHub Webhook
+title: GitHub Webhook Connector
```

Comment:
All but a few connector pages include `Connector` in their title & sidebar names, so I made them all consistent.